apps/client/src/features/app-settings/quick-start/QuickStart.tsx

```diff
@@ -92,30 +92,34 @@ export default function QuickStart(props: QuickStartProps) {
                   description='Default time format to show in views 12 /24 hours'
                   error={errors.settings?.timeFormat?.message}
                 />
-                <Select variant='ontime' size='sm' width='auto' isDisabled={false} {...register('settings.timeFormat')}>
-                  <option value='12'>12 hours 11:00:10 PM</option>
-                  <option value='24'>24 hours 23:00:10</option>
-                </Select>
+                <NativeSelectRoot size='sm'>
+                  <NativeSelectField width='auto' {...register('settings.timeFormat')}>
+                    <option value='12'>12 hours 11:00:10 PM</option>
+                    <option value='24'>24 hours 23:00:10</option>
+                  </NativeSelectField>
+                </NativeSelectRoot>
               </Panel.ListItem>
               <Panel.ListItem>
                 <Panel.Field
                   title='Views language'
                   description='Language to be displayed in views'
                   error={errors.settings?.language?.message}
                 />
-                <Select variant='ontime' size='sm' width='auto' isDisabled={false} {...register('settings.language')}>
-                  <option value='en'>English</option>
-                  <option value='fr'>French</option>
-                  <option value='de'>German</option>
-                  <option value='hu'>Hungarian</option>
-                  <option value='it'>Italian</option>
-                  <option value='no'>Norwegian</option>
-                  <option value='pt'>Portuguese</option>
-                  <option value='es'>Spanish</option>
-                  <option value='sv'>Swedish</option>
-                  <option value='pl'>Polish</option>
-                  <option value='zh'>Chinese (Simplified)</option>
-                </Select>
+                <NativeSelectRoot size='sm'>
+                  <NativeSelectField width='auto' {...register('settings.language')}>
```

Comment:
question: it doesnt look like the width property is necessary anymore?

```suggestion
                  <NativeSelectField {...register('settings.language')}>
```

Reply:
Removed in https://github.com/cpvalente/ontime/pull/1438/commits/d06d5bf80ff52a12ad862541d82a77da51f4c856